test/image/mocks/pseudo_html.json

```diff
@@ -3,7 +3,7 @@
   {
    "x": ["<b>b</b>   <i>i</i>", "line <em>1</em><br>line <b>2</b>"],
    "y": ["sub<sub>1</sub>", "sup<sup>2</sup>"],
-   "name": "<b>test</b> <i>pseudo</i>HTML<br><sup>3</sup>H<sub>2</sub>O is <em>heavy!</em><br>and <span style=\"color:#f00;font-family:'Times New Roman', Times, serif\">Fonts,</span><br>oh my?"
+   "name": "<b>test</b> <i>pseudo</i>HTML<br><sup>3</sup>H<sub>2</sub>O is <em>heavy!</em><br>and <span style=\"color:#f00;font-family:'Times', serif\">Fonts,</span><br>oh my?"
```

Comment:
Is this change necessary? Would the former family list be handled differently by the new code?

Reply:
If I remember correctly (it's been a while now), it caused errors with the test automation when run by CircleCI (don't have exact details at the moment). I assumed it was an environment/setup issue, for example, font not installed.  Removing Times New Roman helped get the test case to run and pass without errors. I believe the spirit of the test case is intact to show that the font style did get applied as expected.

Reply:
hmph, the CI env hasn't changed though, so if the same input that used to work now causes errors, doesn't that mean there's something different now about how we're handling these attributes?

Overall this is a fantastic change! Just want to make sure we aren't creating a subtle bug along the way.

Reply:
Thanks!  Yes, it makes sense to double check this change.  I can try to reproduce when I get a chance (maybe in the next week or two).  I vaguely remember it causing the CircleCI tests to fail, but no errors when testing the fonts via the browser on my laptop.  I can try to verify with a CodeSandbox.